modules/log-analytics-workspace/vars.tf

```diff
@@ -14,9 +14,9 @@ variable "resource_group_name" {
 }
 
 variable "sku" {
-  description = "The SKU that determines the class of workspace to be provisioned. Currently only `PerGB2018` is accepted for new workspaces."
+  description = "Specifies the SKU of the Log Analytics Workspace."
   type        = string
-  default     = "PerGB2018"
+  default     = null
 }
```

Comment:
`PerGB2018` is already the default value.
Passing `null` ensure that this picks the terraform default for this module, so if terraform changes the wordings for `PerGB2018` to `PerGB2021`, then we won't have any issues.
It might be good to avoid hard-coding so far down in modules, and keep it as generic as possible.
`infrastructure-modules` on the other hand can be more opinionated.